src/EiNoah.ts

```diff
@@ -165,7 +165,7 @@ class EiNoah {
             return;
           }
 
-          msg.channel.startTyping(10000).catch(() => { });
+          msg.channel.startTyping().catch(() => { });
           const em = orm.em.fork();
```

Comment:
dit is de fix?

Reply:
Nee de fix is de resolve(null) sinds de Promise nooit resolved werd of rejected werd, werd de finally nooit getriggered